mod_bot/main.py

```diff
@@ -213,6 +213,24 @@ def add_minutes(hstart, mstart, mn):
         await message.channel.edit(name=new_name)
         return
 
+    if words[0] == '!time' and len(words) > 1 and 'fin' in message.channel.name:
+        cwords = message.channel.name.split('-')
+        new_time = None
+        for fmt in ('%Hh%M', '%H:%M'):
+            try:
+                if new_time is None:
+                    new_time = datetime.datetime.strptime(words[1], fmt).time()
+            except ValueError:
+                pass
+        if new_time is not None:
+            new_name = '-'.join(cwords[:-1] + [new_time.strftime('fin%Hh%M')])
+            await message.channel.edit(name=new_name)
+            await message.channel.send('@here Heure de fin de raid mise à jour : ' + new_time.strftime('%Hh%M'))
```

Comment:
Not sure about the @here in the message, should we really mention every one online because of the change ?

Reply:
true, however do you think we should at least mention the participants?

Reply:
@nicolasblaye sorry for the late reply ; mentionning the participants seems to be a good compromise, but "outsiders" may not see the change ; let's leave the @here first and we'll edit it later if users are not happy (or even better, some kind of "opt-in" where administrators can choose if it mentions people or not)